backend/app/main.py

```diff
@@ -2,7 +2,7 @@
 import typing
 
 from fastapi import FastAPI, Request
-from fastapi.middleware.cors import CORSMiddleware
+from starlette.middleware.cors import CORSMiddleware
```

Comment:
I think the old and the new package are actually the same (since FastAPI is built on Starlette); but `main` shows the `fastapi.middleware.cors` module path, so I don't think there's any reason to change it (back).